docs/implementation-status.md

```diff
@@ -0,0 +1,245 @@
+# Implementation Status
+
+Caffeine is an implementation of the Parallel Runtime Interface for Fortran (PRIF). This document
+outlines the implementation status in Caffeine of the features defined in the
+[latest PRIF revision](https://dx.doi.org/10.25344/S4WG64). Caffeine contains interfaces for all
+of the PRIF procedures and are callable, but some procedure definitions are unimplemented. For
+more details about the implementation of the various PRIF features, please see the
+following sections:
+
+- [Named Constants](#Named-Constants)
+- [`stat` and `errmsg` support](#stat-and-errmsg-support) - NO support yet
+- [Program Startup and Shutdown](#Program-Startup-and-Shutdown)
+- [Image Queries](#Image-Queries)
+- [Storage Management](#Storage-Management)
+- [Coarray Queries](#Coarray-Queries)
+- [Continguous Coarray Access](#Continguous-Coarray-Access)
+- [Strided Coarray Access](#Strided-Coarray-Access)
+- [SYNC Statements](#SYNC-Statements)
+- [Locks and Unlocks](#Locks-and-Unlocks)
+- [Critical](#Critical)
+- [Events and Notifications](#Events-and-Notifications)
+- [Teams](#teams)
+- [Collectives](#Collectives)
+- [Atomic Memory Operations](#Atomic-Memory-Operations)
+
+TODO: For features with all no support, flesh out table, but comment it out
+
+## Named Constants
+
+Caffeine contains definitions for all of the PRIF-relevant constants from ISO_FORTRAN_ENV and for
+all of the PRIF-specific constants.
+
+## `stat` and `errmsg` support
+
+Many PRIF procedures have optional arguments `stat`, `errmsg`, and `errmsg_alloc`. There is no
+support for these optional arguments in Caffeine yet and this is not represented in the tables
+in the following sections.
+
+
+## Program Startup and Shutdown
+### Support = partial
+
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_init` | yes | n/a |
```

Comment:
Looking over the rendered doc, the repeated `n/a` in the Notes column is somewhat visually distracting and makes the tables look "busier" than they are. I suggest we just leave the Notes cell blank when we have nothing to say.

Reply:
Great point. I have removed the `n/a` notes from the tables.